### PR TITLE
chore: Add PROBES_PORT environment variable and update targetPorts for existing services

### PR DESCRIPTION
**Pull request by kahirokunn**
# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by lbernick** [comment]
/ok-to-test
@kahirokunn thanks for your PR! Could you please add a commit message body/PR description explaining the reason for these changes? I wouldn't expect this to be user facing, so no release note is needed.

File: config/webhook.yaml [hunk 1]
```diff
@@ -101,6 +101,10 @@ spec:
           value: config-leader-election
         - name: CONFIG_FEATURE_FLAGS_NAME
           value: feature-flags
+        # If you change PROBES_PORT, you will also need to change the
+        # containerPort "probes" to the same value.
+        - name: PROBES_PORT
+          value: "8443"
```
Comment (Yongxuanzhang) [inline config/webhook.yaml:107]: Why this is 8843, in current code it is 8080
https://github.com/tektoncd/pipeline/blob/c20c2000fa04253c3ac126304746d66871dcf878/cmd/webhook/main.go#L252-L255
Are they the same thing?

**Review by Yongxuanzhang — COMMENTED** [review]
the integrations are failing, not sure if it is due to the port setting? 
And could you add PR description as well? Thanks!

File: config/webhook.yaml (hunk 1 above)
Comment (kahirokunn) [reply config/webhook.yaml]: thx. I fixed.

File: config/webhook.yaml [hunk 2]
```diff
@@ -179,15 +184,16 @@ spec:
   # Define metrics and profiling for them to be accessible within service meshes.
   - name: http-metrics
     port: 9090
-    targetPort: 9090
+    targetPort: metrics
```
[the remark this answers is not in the document]
Comment (kahirokunn) [reply config/webhook.yaml:187]: The targetPort of the metrics is defined here
https://github.com/tektoncd/pipeline/blob/20b9f9a67cfe9a64f9d7acd36cc52c9e76e05d6c/config/webhook.yaml#L139

**Comment by kahirokunn** [comment]
@lbernick 
I described the reason in the commit message.
How about now?

**Comment by lbernick** [comment]
It sounds like you are trying to change the probes port value for the webhook on your cluster post-installation and I'm curious why? what is probes port used for?

**Comment by kahirokunn** [comment]
@lbernick

There are cases like the following:

1. EKS worker clusters are built with vxlan (e.g., cilium)
2. Control planes are outside of vxlan (e.g., EKS)

In such configurations, to pass a webhook from the control plane to a Pod, one of the following procedures needs to be performed:

1. Enable hostnetwork for the Pod
2. Use a NodePort Service

If you choose `1. Enable hostnetwork for the Pod`, all ports will be exposed on the node, causing port conflicts, and you will need to manage the ports of all Pods.

However, even if it is not such a case, it is less confusing for the reader to either fully express all environment variables or not express any at all in the yaml, rather than just partially expressing which environment variables can be changed with some settings.

**Comment by lbernick** [comment]
SGTM! would you mind adding that context to your commit message and pr description? Thanks!
/approve

**Comment by Yongxuanzhang** [comment]
There should be a release section in your PR description, you could check out other PRs and add it to remove the `release-note-label-needed`

**Comment by kahirokunn** [comment]
@lbernick What about the current commit message?

**Comment by kahirokunn** [comment]
@Yongxuanzhang thx. done

**Comment by lbernick** [comment]
commit message looks great, thanks!

**Comment by Yongxuanzhang** [comment]
Thanks!
/lgtm

**Pull request merged**